### DISCUSSION TITLE
Which communication channel to use?

### DISCUSSION THREAD

**Discussion by jahow** [geonetwork/geonetwork-ui#509] [Polls]
There have been several people asking for a way to communicate easily with the growing GeoNetwork-UI community and discuss specific topics such as architecture, roadmap, technical issues and contributions.

**Comment by jahow** [comment]
@davidblasby @alexarobu @archaeogeek please give us your opinion, thanks a lot!

**Comment by fgravin** [comment]
Thanks @jahow 
I was wondering @ticheler if we could have some money from the OSGeo budget to pay a slack ?
Slack would be the best, but the history limit is a pain though.

Cheers

**Comment by edevosc2c** [comment]
Voting IRC on https://libera.chat (so that we can have a bridge Matrix too) for the "Other" option.

geOrchestra already uses IRC and it's working great. No need to create an account on IRC, just type a username and you can't instantly start discussing! IRC is also very popular amongst open source communities.

**Reply by jahow** [reply]
Would it make sense to have a "central" channel based on IRC, and then people can access that using whatever app they use every day (slack, teams, whatever) with Matrix?

**Reply by edevosc2c** [reply]
that's too complicated to setup and maintain. I was talking about the matrix bridge because it's maintained by the guys at libera.chat

**Comment by jahow** [comment]
Eventually it was decided to use Gitter. A room has been created for the project: https://app.gitter.im/#/room/#geonetwork_geonetwork-ui:gitter.im

**Reply by edevosc2c** [reply]
Available at this address for people that have a matrix.org account: https://app.element.io/#/room/#geonetwork_geonetwork-ui:gitter.im
